tests/ducktape/test_producer.py

```diff
@@ -0,0 +1,216 @@
+"""
+Ducktape test for Confluent Kafka Python Producer
+Assumes Kafka is already running on localhost:9092
+"""
+import time
+from ducktape.tests.test import Test
+from ducktape.mark import matrix, parametrize
+
+from tests.ducktape.services.kafka import KafkaClient
+
+try:
+    from confluent_kafka import Producer, KafkaError
+except ImportError:
+    # Handle case where confluent_kafka is not installed
+    Producer = None
+    KafkaError = None
+
+
+class SimpleProducerTest(Test):
+    """Test basic producer functionality with external Kafka"""
+    
+    def __init__(self, test_context):
+        super(SimpleProducerTest, self).__init__(test_context=test_context)
+        
+        # Set up Kafka client (assumes external Kafka running)
+        self.kafka = KafkaClient(test_context, bootstrap_servers="localhost:9092")
+        
+    def setUp(self):
+        """Set up test environment"""
+        self.logger.info("Verifying connection to external Kafka at localhost:9092")
+        
+        if not self.kafka.verify_connection():
+            raise Exception("Cannot connect to Kafka at localhost:9092. "
+                          "Please ensure Kafka is running.")
+        
+        self.logger.info("Successfully connected to Kafka")
+        
+    def test_basic_produce(self):
```

Comment:
Let's add a `test_produce_with_schema_registry()` to we can benchmark test with SR 👍